control-plane/agents/src/bin/core/volume/operations.rs

```diff
@@ -1054,3 +1065,56 @@ impl CreateVolumeExeVal for CreateVolumeSource<'_> {
         }
     }
 }
+
+// spread_label_is_same checks if the node labels is same as that of all existing replicas.
+fn spread_label_is_same(
```

Comment:
I think this should "live" in the scheduling modules and not here.

Reply:
tried putting it scheduling but it errored out 
```
error[E0446]: restricted type `volume::operations::Context<'_>` in public interface
   --> control-plane/agents/src/bin/core/volume/scheduling.rs:149:1
    |
149 | / pub(crate) fn spread_label_is_same(
150 | |     replica: &Replica,
151 | |     replica_candidate: &CreateReplica,
152 | |     context: &Context,
153 | | ) -> Result<bool, SvcError> {
    | |___________________________^ can't leak restricted type
    ```